control-plane-operator/controllers/hostedcontrolplane/snapshotcontroller/assets/05_operator_role-hypershift.yaml

```diff
@@ -67,6 +67,13 @@ rules:
   - create
   - patch
   - update
+- apiGroups:
+  - ""
+  resources:
```

Comment:
Does this need access to all secrets in the namespace, or can we narrow this down to specific resource names?

Reply:
This is only a role (as opposed to clusterrole), so only applies to the namespace where it lives

Reply:
@csrwng I know. My question still stands -- does this role require access to all secrets in the namespace, or if it's looking at one specific secret (the description says `csi-snapshot-webhook-secret`) then can we add a resource name to scope the access to that?

Reply:
That's how Informers (https://github.com/openshift/cluster-csi-snapshot-controller-operator/pull/157/commits/277099195090e89d87b2e07228ef679e70e1d771#diff-0d623dfd885adb20f991bda4c2453aebd732ca6dbb4d1d4be6e79805c3b48de6R259) work. We can narrow down list-n-watch to specific namespace, but not to specific secret in namespace. I don't know if it's implementation restriction of Informers or API server itself.